addons/sale_mrp/models/account_move.py

```diff
@@ -3,6 +3,20 @@
 from odoo import models
 
 
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def get_product_and_components(self, product):
+        product_and_components = super().get_product_and_components(product)
+        bom = product.product_tmpl_id.bom_ids and product.product_tmpl_id.bom_ids[0]
```

Comment:
use `bom_find` https://github.com/odoo/odoo/blob/14.0/addons/mrp/models/mrp_bom.py#L173

Reply:
Thanks

Reply:
@fmdl can you please review it again?